tools/ci/lua_stylecheck.py

```diff
@@ -94,7 +94,7 @@ def check_parameter_padding(self, line):
         """
         # ,[^ \n] : Any comma that does not have space or newline following
 
-        for _ in re.finditer(",[^ \n]", line):
+        for _ in re.finditer(",[^ \n\"]", line):
```

Comment:
@claywar can you verify this doesn't do anything spicy?

Reply:
Also, the comment above this will need to be updated

Reply:
This breaks CI causing the case of `(param1,"param2)` to pass
![image](https://github.com/LandSandBoat/server/assets/81713309/0bb5ba3a-c718-4671-b408-8ba4d8378eff)

Reply:
Update: I see where your issue was, and corrected this in #4128